### ISSUE TITLE
Single Quotes in Marker Popups Break HTML

### ISSUE THREAD

**Issue by crunchcrackle** [python-visualization/folium#766]
```python
map_1 = folium.Map(location=[-23.5505, -46.6333],
                   zoom_start=12)

folium.Marker([-23.550519999999999, -46.633308999999997], popup = 'name').add_to(map_1)
folium.Marker([-23.5505, -46.6333], popup = "name'with'quotes").add_to(map_1)

map_1.save('marker_name_test.html')
```
#### Problem description
If a marker popup string contains single quotes, HTML parsing in web browsers breaks and the output HTML file can't be rendered.

#### Expected Output
A correctly rendered HTML page with the single quotes properly escaped or stripped from the popup string.

#### Output of ``folium.__version__``
'0.5.0'

**Comment by ocefpaf** [comment]
Folium expects `html` for the popup text by default to make it easier for user to pass complex popups.

If you want `folium` to transform that to `HTML` for you use the `parse_html=True`. See cell [3] from http://nbviewer.jupyter.org/github/python-visualization/folium/blob/master/examples/Popups.ipynb

**Comment by crunchcrackle** [comment]
I see, my bad!

**Comment by boileaum** [comment]
I spent time trying to understand the origin of this problem. I was running folium from a jupyter notebook. The web browser JavaScript console on the generated html was the only way I found to show the error. I would expect an error to be raised by `folium.Map()` when this happens.

**Comment by crunchcrackle** [comment]
I expected an error to be raised as well. In my situation, I had to open up the dev console in Chrome to figure out what the error was.

**Comment by MCOfficer** [comment]
seeing this error pop up over and over again on SO, i'd recommend at least printing a warning if the html contains an unescaped single quote.

**Comment by Conengmo** [comment]
Thanks for the comment @MCOfficer. Thankfully this issue has been fixed as far as I know in #962, which will be in the next release.
